libraries/tests/mbed/pwm/main.cpp

```diff
@@ -19,15 +19,15 @@ float value = 0.75;
 int main() {
 #if defined(TARGET_FF_ARDUINO)
     PwmOut pwm(D9);
+    int period_us = 40000; // allows to test various periods
```

Comment:
Can you provide details for this test change? Changing period from 10 ms to 40 000 us? 

The commit msg says "Update ARDUINO test to allow period change". How do you expect to change the freq - manually ?